android-components/components/feature/downloads/src/main/java/mozilla/components/feature/downloads/share/ShareDownloadFeature.kt

```diff
@@ -144,10 +144,15 @@ class ShareDownloadFeature(
         scope?.launch(coroutineExceptionHandler) {
             withTimeout(OPERATION_TIMEOUT_MILLIS) {
                 val download = download(internetResource)
+                val shareMessage = if (internetResource.contentType != null) {
+                    sanitizeLongUrls(internetResource.url)
+                } else {
+                    null
+                }
                 share(
                     contentType = internetResource.contentType,
                     filePath = download.canonicalPath,
-                    message = sanitizeLongUrls(internetResource.url),
+                    message = shareMessage,
```

Comment:
I think this message is what we needed to remove and so we can leave it as `null` by default?
Can you check with sharing an image to gmail?

Reply:
Gmail video:

https://user-images.githubusercontent.com/32488956/208458965-494f0b4a-edc9-481a-bd5b-c0c393ad2d88.mp4